build/webpack.common.js

```diff
@@ -5,11 +5,10 @@
 const path = require('path');
 const { webpackAliases } = require('../utils/tooling/aliasHelper.js');
 const { PRODUCTION, DEVELOPMENT } = require('../utils/tooling/constants.js');
-const babelPresets = require('./babelPresets.js');
 
 // constants
 const UI_TITLE = 'Strimzi UI';
-const BUILD_DIR = path.resolve(__dirname, '../dist');
+const BUILD_DIR = path.resolve(__dirname, '../dist/client');
```

Comment:
I may spot this later, but what was the motivation for this change?

Reply:
To have the client code output in a different directory than the server.  Figured we will output the server to ../dist/server